src/modules/dashboard/components/UnlockTokenDialog/UnlockTokenForm.tsx

```diff
@@ -92,6 +97,11 @@ const UnlockTokenForm = ({
             href={LEARN_MORE_URL}
           />
         </div>
+        <Annotations
+          label={MSG.annotation}
```

Comment:
There's a case to make here that the core `Annotation` component should have a default label value, since we're basically using the same text for all modals.

This way we could cut down on some code repetition

Reply:
There are 3 instances of the repeated message.

> Explain why you’re making these changes (optional)

This could be the default message, and also still have the customisable message, for cases where it is more descriptive. Such as:

> Explain why you’re editing the colony's details (optional)

Q. Would you like me to make the Unlock Token message more descriptive? And / or include the default message changes in this PR?